artiq/firmware/runtime/moninj.rs

```diff
@@ -55,7 +55,7 @@ mod remote_moninj {
     use sched::{Io, Mutex};
 
     pub fn read_probe(io: &Io, aux_mutex: &Mutex, linkno: u8, destination: u8, channel: u16, probe: u8) -> u32 {
-        let reply = drtio::aux_transact(io, aux_mutex, linkno, &drtioaux::Packet::MonitorRequest { 
+        let reply = drtio::aux_transact(io, aux_mutex, linkno, &drtioaux::Packet::MonitorRequest {
```

Comment:
Remove this change.

Reply:
seems like this line is not formatted properly, the space shouldn't be there

Reply:
As I told you multiple times already, style-only changes should not be bundled with other things.